Core/src/main/java/com/paypal/android/core/PaymentsConfiguration.kt

```diff
@@ -2,5 +2,6 @@ package com.paypal.android.core
 
 data class PaymentsConfiguration(
     val clientId: String,
-    val environment: Environment = Environment.SANDBOX
+    val environment: Environment = Environment.SANDBOX,
+    val authToken: String? = null
```

Comment:
Is `authToken` needed here? I think we can rely on the `clientId` for authorization.

Reply:
We realized that we can also use `authtoken` to make the request. This is just here for testing, because the request accepts encoded: `<client_id>:<secret>` or the `authtoken`. I think we need to some global variables to the sample app, that the merchant can input his `secret` and `clientId`.

Reply:
Yeah and IMO moving away from client secret as soon as possible is best. We'll decrease the risk of committing one of our test merchant's secret into GitHub. We implemented a script to fetch access-tokens in this PR for convenience.

Reply:
@tdchow we also tried `Basic ${base64(clientId)}` auth and got `4xx` responses.

Reply:
Got it. Currently the API will not accept the `Basic ${base64(clientId)}` header. Hopefully, the API team wraps up the changes to properly accept this auth header. Can we add a comment to call out that `authToken` is currently used for testing and that it'll be removed after the API changes are made?

Reply:
Went ahead and added basic auth with `base64(CLIENT_ID:CLIENT_SECRET)`. We won't need to mention auth token going forward.